PowerEditor/src/Notepad_plus.cpp

```diff
@@ -2516,6 +2516,20 @@ int Notepad_plus::findMachedBracePos(size_t startPos, size_t endPos, char target
 	}
 	else // forward - TODO
 	{
+		int balance = 0;
+		for (int i = (int)startPos; i <= (int)endPos; ++i) {

```

Comment:
1. for type conversion use C++-style as stated in the [Contributing Guideline](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md)
2. or change to `for (int i = int(startPos); i <= int(endPos); ++i) {` like in `backward` branch.